addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -739,13 +761,57 @@ QUnit.test('use a command', async function (assert) {
     );
 });
 
+QUnit.test("commands are not displayed when channel types don't match", async function (assert) {
```

Comment:
_''channel with no commands should not prompt any command suggestions on typing /"_

Reply:
Done